rules-tests/TypeDeclaration/Rector/ClassMethod/NarrowObjectReturnTypeRector/Fixture/skip_multiple_return_types.php.inc

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Rector\Tests\TypeDeclaration\Rector\ClassMethod\NarrowObjectReturnTypeRector\Fixture;
+
+class Product
+{
+}
+
+class Service
+{
+}
+
+final class MixedFactory
+{
+    public function create(): object
+    {
+        if (rand(0, 1)) {
+            return new Product();
+        }
+
+        return new Service();
```

Comment:
no need to create dummy classes, just use `\DateTime` and `\stdClass` is enough to demonstrate this.